Altis_Life.Altis/core/functions/fn_startLoadout.sqf

```diff
@@ -6,23 +6,44 @@
     Description:
     Loads a custom loadout on player when he got a new life
 */
+private _side = call {
+    if (playerSide isEqualTo civilian) exitWith {"civ"};
+    if (playerSide isEqualTo west) exitWith {"cop"};
+    if (playerSide isEqualTo independent) exitWith {"med"};
+};
+
 private _level = call {

```

Comment:
Bit of duplication with line 31 here I think. I think we could do it better by doing something like this:
`
if (_side isEqualTo "civ") then {
    if (life_is_arrested) then {
        //arrestedLoadout
    } else {
        //randomLoadout
    };
} else {
    //fetchRelevantLevel
    //setLoadout  
};`